### PR TITLE
Add Arch Linux install instructions

### PR DESCRIPTION
**Pull request by ISSOtm**
I didn't test the devkitARM install instructions, but they are pretty much transcribed from the Ubuntu/Deb ones, so that should be fine.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -305,6 +305,20 @@ Then proceed to [Choosing where to store pokeemerald (Linux)](#choosing-where-to
 >   then you will have to install devkitARM. Install all the above packages except binutils-arm-none-eabi, and follow the instructions to
 >   [install devkitARM on Debian/Ubuntu-based distributions](#installing-devkitarm-on-debianubuntu-based-distributions).
 </details>
+    
+### Arch Linux
+Run this command as root to install the necessary packages:
+```console
+# pacman -S base-devel arm-none-eabi-binutils git
```
Comment (GriffinRichards) [inline INSTALL.md:312]: Exclude the `#` here and below, beginners will use these instructions and likely include it if they're copy-pasting

Comment (ISSOtm) [reply INSTALL.md:312]: D'aw, I thought GitHub would do the reasonable thing and mark the `console` prompt as un-selectable. --'

**Comment by ISSOtm** [comment]
Fixed the above feedback, as well as a missing `libpng` package install. Thanks @SatoMew!

**Comment by SatoMew** [comment]
> Thanks @SatoMew!

No worries! 😄

**Comment by GriffinRichards** [comment]
The `#` in `# pacman -S gba-dev` is still present.

**Comment by ISSOtm** [comment]
Fixed that one as well, yeah.

**Pull request merged**